app/src/main/java/com/sdp13epfl2021/projmag/database/UserDataDatabase.kt

```diff
@@ -81,4 +80,33 @@ interface UserDataDatabase {
         onSuccess: () -> Unit,
         onFailure: (Exception) -> Unit
     )
-}
\ No newline at end of file
+
+    /**
+     * Apply to the given project by pushing it to
+     * Call `onSuccess` if the operation succeeded
+     * Call `onFailure` with an Exception in case of failure
+     *
+     * @param applied function applies if true, else unapplies
+     * @param onSuccess called on success
+     * @param onFailure called with an exception on failure
+     */
+    fun applyUnapply(
+        applied: Boolean,
```

Comment:
So, if applied is true it should add the projectId,
and if applied is false it will remove it ?

Reply:
I changed the variable to apply in the implementation of the interface but forgot to change it here

Reply:
Now it's the documentation that is not the same as the function.
And I'm still not sure that I understood what this function should do when it's true/false.

Reply:
Indeed you are right, I fixed it.